packager/react-packager/src/DependencyResolver/Module.js

```diff
@@ -15,7 +15,7 @@ const replacePatterns = require('./replacePatterns');
 
 class Module {
 
-  constructor(file, fastfs, moduleCache, cache) {
+  constructor(file, fastfs, moduleCache, cache, helpers) {
```

Comment:
we should probably change this into taking a single object, and use destructuring in here.